Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -261,8 +261,10 @@ def create_indicators_from_response(raw_response, tlp_color=None, feed_tags=None
         indicator: dict = {}
 
         for resource in raw_response['resources']:
+            if not (type_ := auto_detect_indicator_type_from_cs(resource['indicator'], resource['type'])):
+                continue
```

Comment:
let's log here - `f"unexpected {type=} for {resource.get('indicator', '')}, skipping."`